### ISSUE TITLE
begin_end_names: dont report none if there are unknown

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3282]
```abap
  TYPES: BEGIN OF something,
           something TYPE STANDARD TABLE OF  WITH DEFAULT KEY,
         END OF something.
```
